digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/model/RoadAddressLinkPartitioner.scala

```diff
@@ -6,7 +6,11 @@ object RoadAddressLinkPartitioner extends GraphPartitioner {
 
   def partition(links: Seq[RoadAddressLink]): Seq[Seq[RoadAddressLink]] = {
     val linkGroups = links.groupBy { link => (
-      link.anomaly.value, link.roadNumber, link.roadPartNumber, link.trackCode, link.roadLinkType.value
+      /*
+      Ignores the roadLinkType from the linkGroups, to the task 245 -> complementary roads
+      */
+      //link.anomaly.value, link.roadNumber, link.roadPartNumber, link.trackCode, link.roadLinkType.value
```

Comment:
Is this commented out code needed somewhere, because If it isn't please do remove it.